### DISCUSSION TITLE
Danish characters not allowed in team name

### DISCUSSION THREAD

**Discussion by Znegl** [vercel/vercel#3693] [Ideas]
When creating a new team I'm not allowed to enter the real name for the team because it contains a danish character.
The same bug might apply for other special characters.
The name entered is `FDF Nivå`
![Screenshot 2020-01-29 at 10 23 31](https://user-images.githubusercontent.com/108805/73343834-88b50300-4281-11ea-9581-33f55eab7444.png)

**Comment by styfle** [comment]
We are working on expanding the permitted characters for team names, thanks!
